cmd/update.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"github.com/blang/semver"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var (
+	githubSlug     = "screwdriver-cd/sd-local"
+	currentVersion = version
+	updateFlag     = false
+)
+
+func canUpdate() (*selfupdate.Release, error) {
+
+	if currentVersion == "dev" {
+		err := errors.New("This is a development command and cannot be updated.")
+		return &selfupdate.Release{}, err
+	}
+
+	latest, found, err := selfupdate.DetectLatest(githubSlug)
+	if err != nil {
+		return &selfupdate.Release{}, err
+	}
+	if !found {
+		err = errors.New("Repositry Not Found")
+		return &selfupdate.Release{}, err
+	}
+	v := semver.MustParse(currentVersion)
+	if latest.Version.LTE(v) {
+		err = errors.New("Current version is latest")
+		return &selfupdate.Release{}, err
+	}
+	return latest, err
```

Comment:
`err` should be nil?
```suggestion
	return latest, nil
```